clayui.com/gatsby/createPages.js

```diff
@@ -95,17 +95,17 @@ const createDocs = (actions, edges, mdx, blacklist = []) => {
 					const blacklistSlugs = blacklist.map(
 						({
 							node: {
-								fields: {sibling},
+								fields: {siblings},
 							},
-						}) => sibling
+						}) => siblings
 					);
 
 					createPage({
 						component: template,
 						context: {
 							blacklist: blacklistSlugs,
 							markdownJsx: mdx,
-							sibling,
+							siblings,
 							slug,
 						},
```

Comment:
Try to normalize the `blacklistSlugs` variable to be a flat array, it should now return something like this:

```
[
  ['path', 'path/some/'],
  ['path']
]
```

As we are passing this on to the template, we want a `Array<String>`.